internals/search.py

```diff
@@ -80,6 +99,8 @@ def process_query(user_query):
     return process_starred_me_query()
   if user_query == 'owner:me':
     return process_owner_me_query()
+  if user_query == 'is:recently-reviewed':
```

Comment:
Are these queries documented anywhere or it is set by the client side?

Reply:
For now, each query is hard-coded into the client to populate specific boxes in the UI.  

We might generalize this to a user query language at some point in the future.  The fact that these queries are not visible to users means that they do not need end-user documentation and we could rework the query language without users needing to relearn anything.